### PR TITLE
Sidecar upgrade for livenessprobe & snapshotter in 2.10.0

### PR DESCRIPTION
**Pull request by Jainbrt**
## Pull request checklist

```release-notes

```

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## What is the current behavior?

-

## What is the new behavior?

-

## How risky is this change?
- [ ] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Review by deeghuge — APPROVED** [review]

File: operator/config/overlays/cnsa/kustomization.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ patches:
                   - name: CSI_PROVISIONER_IMAGE
                     value: cp.icr.io/cp/spectrum/scale/csi/csi-provisioner@sha256:893e37a388e7a7463d6c3523311b28cfbc5ae536dbef35430eed272cdc6850dc
                   - name: CSI_LIVENESSPROBE_IMAGE
-                    value: cp.icr.io/cp/spectrum/scale/csi/livenessprobe@sha256:2b10b24dafdc3ba94a03fc94d9df9941ca9d6a9207b927f5dfd21d59fbe05ba0
+                    value: cp.icr.io/cp/spectrum/scale/csi/livenessprobe@sha256:4dc0b87ccd69f9865b89234d8555d3a614ab0a16ed94a3016ffd27f8106132ce
```
Comment (amdabhad) [inline operator/config/overlays/cnsa/kustomization.yaml:39]: Before merging the PR, this image has to be pushed to artifactory. Or we can revert the change in cnsa/kustomization.yaml and make this change only on the day of build delivery.

**Review by amdabhad — CHANGES_REQUESTED** [review]

File: operator/config/overlays/cnsa/kustomization.yaml (hunk 1 above)
Comment (Jainbrt) [reply operator/config/overlays/cnsa/kustomization.yaml:39]: True, I am already checking with @hemalathagajendran to push these images to artifactory.

**Review by amdabhad — APPROVED** [review]
